pkg/virt-controller/watch/application.go

```diff
@@ -606,6 +606,10 @@ func (vca *VirtControllerApp) initCommon() {
 			}),
 		services.WithSidecarCreator(
 			func(vmi *v1.VirtualMachineInstance, kvc *v1.KubeVirtConfiguration) (hooks.HookSidecarList, error) {
+				if err := netbinding.ValidateVMINetBindingPlugins(vmi, kvc); err != nil {
```

Comment:
Why is this validation part of the `WithSidecarCreator`?

Reply:
This is where the NetBindingPluginSidecarList is called, so I thought it would  make sense to call it right next to it comparing to call `ValidateVMINetBindingPlugins` in other place, probably higher in the stack.